src/js/project/CreateProjectWizard.jsx

```diff
@@ -436,6 +332,7 @@ export default class CreateProjectWizard extends React.Component {
 
   tryChangeStep = (newStep, alertUser = true) => {
     const errorList = this.getSteps()[this.context.wizardStep].validate();
+    console.log("project/CreateProjectWizard/tryChangeStep errorList: ", errorList);
```

Comment:
Do we need this console.log? If we do, I feel like we could remove the path to the file.